jakartaee-api/pom.xml

```diff
@@ -158,36 +146,47 @@
 
         <!-- Optional APIs -->
         <dependency>
-            <groupId>jakarta.enterprise.deploy</groupId>
-            <artifactId>jakarta.enterprise.deploy-api</artifactId>
-            <version>${jakarta.enterprise.deploy-api.version}</version>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <version>${jakarta.xml.bind-api.version}</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.xml.ws</groupId>
+            <artifactId>jakarta.xml.ws-api</artifactId>
+            <version>${jakarta.xml.ws-api.version}</version>
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>jakarta.xml.registry</groupId>
-            <artifactId>jakarta.xml.registry-api</artifactId>
-            <version>${jakarta.xml.registry-api.version}</version>
+            <groupId>jakarta.jws</groupId>
+            <artifactId>jakarta.jws-api</artifactId>
+            <version>${jakarta.jws-api.version}</version>
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>jakarta.xml.rpc</groupId>
-            <artifactId>jakarta.xml.rpc-api</artifactId>
-            <version>${jakarta.xml.rpc-api.version}</version>
+            <groupId>jakarta.xml.soap</groupId>
+            <artifactId>jakarta.xml.soap-api</artifactId>
+            <version>${jakarta.xml.soap-api.version}</version>
             <optional>true</optional>
         </dependency>
 
+
+
         <!-- Compile-time dependencies -->
+        <!-- work around for GLASSFISH-19861  -->
         <dependency>
-            <groupId>jakarta.xml.bind</groupId>
-            <artifactId>jakarta.xml.bind-api</artifactId>
-            <version>${jakarta.xml.bind-api.version}</version>
+            <groupId>org.glassfish</groupId>
+            <artifactId>jakarta.faces</artifactId>
+            <version>${mojarra.version}</version>
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>jakarta.xml.ws</groupId>
-            <artifactId>jakarta.xml.ws-api</artifactId>
-            <version>${jakarta.xml.ws-api.version}</version>
+            <groupId>com.sun.mail</groupId>
+            <artifactId>jakarta.mail</artifactId>
+            <version>${jakarta.mail-api.version}</version>
```

Comment:
Hmm, haven't hit the problem yet.  But, thanks for the heads up.  I'll look into it.

Reply:
@bshannon, Activation looks to be compiling just fine...  Without bringing in the Impl jar...  Are you sure it's required?  I'm not going to make any changes for Activation yet since I can't reproduce the problem.

Reply:
I have to admit that I don't *fully* understand the original problem, but based on
my limited understanding I expected it would be necessary.  If it's not, well that's
great!